+light_python_wrapper/m2p.m

```diff
@@ -46,7 +46,9 @@
             p = m;
     end
     if ~isscalar(m) && ~ischar(m)
-        if verLessThan('matlab','9.5')
+        vers = version();
+        is_old_version = sscanf(vers(1:3), '%f') < 9.5
```

Comment:
Thanks for doing this... it's always little things like these that surprise you - like Jacob found with the struct comparison in Horace...

Anyway, what about making this a `persistent` value and saving a few more seconds...? E.g. put:

```matlab
persistent is_old_version
if isempty(is_old_version)
    vers = version();
    is_old_version = sscanf(vers(1:3), '%f') < 9.5;
end
```

near line 24 and remove line 49-50?

Reply:
Done!